SPIE2016-9913-16/9913-16.tex

```diff
@@ -169,36 +169,53 @@ \subsection{Region handling}
 If Astropy and LSST cannot share sky to pixel region handling code directly there is a possibility that STC could be adopted to allow interoperability.
 
 \begin{figure}[t]
-\centering
-\begin{tabular}{ccc}
-  Discrete Spherical & $\Longleftrightarrow$ & Discrete Cartesian \\
-  $\Updownarrow$ & &  $\Updownarrow$ \\
-  Geometric Spherical & $\Longleftrightarrow$ & Geometric Cartesian \\
-\end{tabular}
-\vspace{10pt}
-\caption{Four distinct types of region. This paper discusses all except discrete spherical.}
+\begin{center}
+\includegraphics[width=0.5\textwidth]{regions}
+\end{center}
+\caption{Four ways of representing regions, and the approximate domains of the LSST (blue) and Astropy (green) packages for manipulating them.}
 \label{fig:regions}
 \end{figure}
 
-Astropy has no immediate plans for a discretized efficient representation of a pixel region, and this is something that LSST can contribute to the Astropy region handling package.
+\subsection{Region handling}
```

Comment:
Yes, sorry.  I actually saw this as a rebase conflict, and thought I'd fixed it; I'm guessing I just forgot to save before doing "git add".